pkg/config/config.go

```diff
@@ -109,6 +113,7 @@ func UpdateConfigFilePath(configFile string) {
 }
 
 func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
+	log.Info("Loading NSX Operator configuration file", "path", configFilePath)
```

Comment:
use lower case for log message? Loading -> loading

Reply:
ok